autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -28,7 +29,7 @@ def __init__(
     ):
         """
         Args:
-            name (str): name of the agent.
+            name (str): name of the agent. It will be used to find the existing assistant by name.
```

Comment:
```suggestion
            name (str): name of the agent. It will be used to find the existing assistant by name. Please remember to delete an old assistant with the same name if you intend to create a new assistant with the same name.
```
Shall we provide a reminder like this? I sense that there will be lots of questions around this.